libs/coin-modules/coin-tezos/src/logic/estimateFees.integ.test.ts

```diff
@@ -56,11 +56,11 @@ describe("estimateFees", () => {
 
     // Then
     expect(result).toEqual({
```

Comment:
Shouldn't we check that account balance minus fees and revelation cost equals 0 ?

Reply:
We might want to do this test on all the coin-module to be consistent.